main.js

```diff
@@ -0,0 +1,50 @@
+// Check if there are any existing books in local storage
+let books = JSON.parse(localStorage.getItem('books')) || [];
+// Function to remove a book from the collection
+function removeBook(book) {
+  books = books.filter((b) => b !== book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+// Function to display all books in the collection
+function displayBooks() {
+  const displayedBooks = document.getElementById('displayedBooks');
+  displayedBooks.innerHTML = '';
+
+  books.forEach((book) => {
+    const div = document.createElement('div');
+    div.innerHTML = `
+      <p>${book.title}</p>
+      <p>${book.author}</p>
+      <button class="removeBtn">Remove</button>
+      <hr>
+    `;
+
+    const removeBtn = div.querySelector('.removeBtn');
+    removeBtn.addEventListener('click', () => {
+      removeBook(book);
+      div.remove();
+    });
+
+    displayedBooks.appendChild(div);
+  });
+}
+
+// Function to add a new book to the collection
+function addBook(title, author) {
+  const newBook = { title, author };
```

Comment:
- [ ] [Optional] Well done on including the `title` and `author` properties in your `books` object as per the project requirement. 👍 Adding an `id` property for each book is a great suggestion to make it easier to identify and remove books with duplicate values.
Here's an example of how you can modify your `books` object to include an `id` property:
```javascript
const books = [
  { id: 1, title: "Book 1", author: "Author 1" },
  { id: 2, title: "Book 2", author: "Author 2" },
  { id: 3, title: "Book 3", author: "Author 3" },
  // ...other books
];
```

By assigning a unique `id` value to each book, you can easily identify and manipulate individual books within the `books` array. Keep up the good work! 👍🌟

Reply:
Thanks!